apps/condo/domains/common/utils/codegeneration/generate.hooks.ts

```diff
@@ -18,29 +18,31 @@ interface IHookConverters<GQL, GQLInput, UI, UIForm> {
 
 interface IHookResult<UI, UIForm, Q> {
     gql: any
-    useObject: (variables: Q, memoize?: boolean) => { obj: UI, loading: boolean, error?: ApolloError | string, refetch?: Refetch<Q> }
-    useObjects: (variables: Q, memoize?: boolean) => { objs: UI[], count: number | null, loading: boolean, error?: ApolloError | string, refetch?: Refetch<Q> }
+    useObject: (variables: Q) => { obj: UI, loading: boolean, error?: ApolloError | string, refetch?: Refetch<Q> }
+    useObjects: (variables: Q) => { objs: UI[], count: number | null, loading: boolean, error?: ApolloError | string, refetch?: Refetch<Q> }
     useCreate: (attrs: UIForm, onComplete: (obj: UI) => void) => (attrs: UIForm) => Promise<UI>
     useUpdate: (attrs: UIForm, onComplete: (obj: UI) => void) => (attrs: UIForm, obj: UI) => Promise<UI>
     useDelete: (attrs: UIForm, onComplete: (obj: UI) => void) => (attrs: UIForm) => Promise<UI>
 }
 
 export function generateReactHooks<GQL, GQLInput, UIForm, UI, Q> (gql, { convertToGQLInput, convertToUIState }: IHookConverters<GQL, GQLInput, UI, UIForm>): IHookResult<UI, UIForm, Q> {
-    function useObject (variables: Q, memoize = true) {
-        const { loading, refetch, objs, count, error } = useObjects(variables, memoize)
+    function useObject (variables: Q) {
+        const { loading, refetch, objs, count, error } = useObjects(variables)
         if (count && count > 1) throw new Error('Wrong query condition! return more then one result')
         const obj = (objs.length) ? objs[0] : null
         return { loading, refetch, obj, error }
     }
 
-    function useObjects (variables: Q, memoize = true) {
+    function useObjects (variables: Q) {
         const intl = useIntl()
         const ServerErrorPleaseTryAgainLaterMsg = intl.formatMessage({ id: 'ServerErrorPleaseTryAgainLater' })
         const AccessErrorMsg = intl.formatMessage({ id: 'AccessError' })
 
         // eslint-disable-next-line prefer-const
         const result = useQuery<{ objs?: GQL[], meta?: { count?: number } }, Q>(gql.GET_ALL_OBJS_WITH_COUNT_QUERY, {
             variables,
+            fetchPolicy: 'no-cache', // Without this flag table will shake on sorting
```

Comment:
It's better to add an extra argument for apollo `useQuery`. 
Example: `useObjects( {}, { fetchPolicy: 'no-cache', ... })`

Reply:
Ok. Add queryOptions to useObjects hook.